contributing/communications.md

```diff
@@ -38,10 +38,10 @@ We don't have _strict_ guidelines on how you should write. We're not going to at
 
 Follow Plain English guidelines to help safeguard the accessibility of your written content. It'd be ironic for an a11y project to produce documents that are hard to understand! All relevant guidelines for [writing well][writing-well] apply. You might find tools like [Hemingway App][hemingway] helpful when editing. In particular:
 
-- Be concise  
+- Be brief  
```

Comment:
![Mind blown](https://media3.giphy.com/media/Es3FISPjOZzAA/giphy.gif "Mind blown")

That does make sense. I'll fix that 😊